### PR TITLE
JPERF-767: Bump log4j to 2.17.0

### PR DESCRIPTION
**Pull request by mzyromski-atlassian**
No dependency locking here.

### CODE REVIEW

**Comment by mzyromski-atlassian** [comment]
There are some issues with Travis CI that I need to sort out first. Most likely I'll try to migrate CI to GHA.

**Comment by mzyromski-atlassian** [comment]
PR to migrate CI to GHA: https://github.com/atlassian/jvm-tasks/pull/11

File: build.gradle.kts [hunk 1]
```diff
@@ -19,7 +19,7 @@ dependencies {
 fun log4j(
     vararg modules: String
 ): List<String> = modules.map { module ->
-    "org.apache.logging.log4j:log4j-$module:2.10.0"
+    "org.apache.logging.log4j:log4j-$module:2.17.0"
```
Comment (dagguh) [inline build.gradle.kts:22]: Dep locks need an update too, right?

**Review by dagguh — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (mzyromski-atlassian) [reply build.gradle.kts:22]: There's no dependency locking in this module.

Comment (dagguh) [reply build.gradle.kts:22]: Roger that :+1:

**Pull request merged**